.github/ISSUE_TEMPLATE/maintainer-onboarding.yml

```diff
@@ -0,0 +1,81 @@
+name: Maintainer Onboarding
+description: Onboarding checklist for maintainers
+title: "Onboard <GitHub handle> as <SIG/WG FOO> maintainer"
+labels: area/github-membership
+body:
+- type: markdown
+  attributes:
+    value: |
+     This issue should be submitted by an existing maintainer of the affected SIG
+- id: sig
+  type: input
+  attributes:
+    label: "SIG name"
+    placeholder: "SIG example"
+  validations:
+    required: true
+- id: maintainers_group
+  type: input
+  attributes:
+    label: "maintainers team"
+    placeholder: "@opentelemetry/<sig-name>-maintainers"
+    description: Tag the maintainers group. If unsure, you can [look groups up](https://github.com/orgs/open-telemetry/teams?query=maintainers)
+  validations:
+    required: true
+- id: github_handle
+  type: input
+  attributes:
+    label: "Github handle of the new maintainer candidate"
+    placeholder: "@example"
+  validations:
+    required: true
+- id: requirements
+  type: checkboxes
+  attributes:
+    label: Requirements
+    description: Before submitting 
+    options:
+      - label: Are you (submitter of this issue) a maintainer of the affected SIG?
+        required: true
+      - label: Is the maintainer candidate a member of the OpenTelemetry GitHub organization already?
+        required: true
+- id: election
+  type: checkboxes
+  attributes:
+    label: Maintainer Election
+    description: |
```

Comment:
Should this point back to the `Becoming a Maintainer` part of the docs rather than repeating the election process?

Reply:
good point! Let me fix that.